frank-flow/src/frontend/src/app/shared/workers/flow-generator.worker.ts

```diff
@@ -33,7 +36,7 @@ const parserWrite = (data: string) => {
 };
 
 parser.on('error', (error) => {
-  console.error(error);
```

Comment:
Alternatively for filtering the errors in the canvas you can also filter the errors here on the line and only add one unique error for each line.

Reply:
I wonder if anyone has had this specific issue with saxes before. Must be a common thing right?